package.json

```diff
@@ -67,27 +69,27 @@
     "styled-components": "^4.1.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.2.2",
+    "@babel/core": "^7.4.5",
     "@babel/preset-flow": "^7.0.0",
     "@integreat-app/eslint-config": "^0.2.1",
     "@integreat-app/eslint-config-jest": "^0.2.1",
     "@integreat-app/eslint-config-react": "^0.3.3",
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-module-resolver": "^3.1.3",
+    "babel-jest": "^24.8.0",
+    "babel-plugin-module-resolver": "^3.2.0",
     "babel-plugin-transform-inline-environment-variables": "^0.4.3",
     "babel-plugin-transform-remove-console": "^6.9.4",
-    "eslint": "^5.12.1",
+    "eslint": "^5.16.0",
     "eslint-config-standard": "^12.0.0",
     "eslint-config-standard-react": "^7.0.2",
-    "eslint-plugin-flowtype": "^3.2.1",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jest": "^22.1.3",
```

Comment:
Are these Upgrades compatible with our eslint packages? yarn shows a Warnung in that case.

Reply:
I didn't see any warnings...

Reply:
You only see them when you remove the `node_modules` dir and rerun flow.

Reply:
If there is no reason for upgrading eslint I would suggest to downgrade them again. I'm not really sure what different version of eslint could cause.

If you feel confident ignore this comment.